core/templates/pages/classroom-page/classroom-page.controller.ts

```diff
@@ -54,7 +54,12 @@ angular.module('oppia').directive('classroomPage', [
             ctrl.bannerImageFileUrl = UrlInterpolationService.getStaticImageUrl(
               '/splash/books.svg');
 
-            PageTitleService.setPageTitle(ctrl.classroomName + ' - Oppia');
+            ctrl.classroomDisplayName = (
+              ctrl.classroomName.charAt(0).toUpperCase() +
+              ctrl.classroomName.slice(1));
+
+            PageTitleService.setPageTitle(
```

Comment:
For the comment in urlservice, page title checking may also be an alternate way.

Reply:
Yeah, that works too, actually, since the page title is derived from the classroomName, we could just use that instead. Though, currently,  the config domain has the TOPIC_IDS_FOR_CLASSROOM_PAGES config property containing classroom names. I'm not sure if these can be accessed in the classroom page controller file. Should I create a constant in constants.ts for classroom names?

Reply:
**Update:** We probably don't need this anymore since I've added the URLs with parameters now, so it would get validated in the handler itself.